src/lib/config.ts

```diff
@@ -250,7 +250,11 @@ class default_config {
         A: "bmark",
         zi: "zoom 0.1 true",
         zo: "zoom -0.1 true",
```

Comment:
dupe?

Reply:
One is shorthand on the commandline `:zo[om]` the other is a keybind.

Reply:
As in, one is an exalias the other a keybind.

Reply:
oh wow. ta.